app/controllers/items_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ItemsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def index
+    @items = Item.all
+  end
+  def new
+    @categories = Category.all
+    @subcategories = SubCategory.all
+    @item = Item.new
+  end
+  def create
+    @item = Item.create(item_params)
+    redirect_to items_path()
+  end
+  def show
+    @item = Item.find(params[:id])
+  end
+  def edit
+    @item = Item.find(params[:id])
+    @categories = Category.all
+    @subcategories =  SubCategory.all
+  end
+  def update
+    @item = Item.find(params[:id])
+    @categories = Category.all
```

Comment:
no need of @categories and @subcategories instance variables

Reply:
removed instance variables